pipelinewise/cli/pipelinewise.py

```diff
@@ -1316,19 +1317,31 @@ def stop_tap(self):
 
                 # Terminate child processes
                 for child in parent.children(recursive=True):
-                    self.logger.info('Sending SIGINT to child pid %s...', child.pid)
-                    child.send_signal(signal.SIGINT)
+                    self.logger.info('Sending SIGTERM to child pid %s...', child.pid)
+                    child.send_signal(signal.SIGTERM)
 
                 # Terminate main process
-                self.logger.info('Sending SIGINT to main pid %s...', parent.pid)
-                parent.send_signal(signal.SIGINT)
+                # self.logger.info('Sending SIGTERM to main pid %s...', parent.pid)
```

Comment:
Can you please remove this commented out code, it hinders readability and maintenance.

Reply:
done